scripts/docker/helpers.bash

```diff
@@ -0,0 +1,7 @@
+function docker_run_routing_release_tests() {
+  local db="${1?Provide DB flavor. e.g e.g mysql-8.0(or mysql),mysql-5.7,postgres}"
+  local this_file_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+  "$this_file_dir/container.bash" "$db" -d
+  docker exec routing-release-docker-container '/repo/scripts/docker/build-binaries.bash'
+  docker exec routing-release-docker-container '/repo/scripts/docker/test.bash'
```

Comment:
Is there ever a case where we'd want to run `test.bash` without first running `build-binaries.bash`?

Reply:
Yeah, not all tests require having the binaries in place. It's best to build-binaries once and have them in docker-container, but if one. wants to run tests for [multi-error](https://github.com/cloudfoundry/multierror/compare/main...with-bin/test.bash) you don't have to build binaries. We did expose the test dependencies so that situation[ like this would not happen](https://github.com/cloudfoundry/routing-acceptance-tests/compare/main...with-bin/test.bash#diff-974b281671c7dc2a0a72f098e7a1547432ebc0216eaf9fe95b46cedad8a6ca6bL24-R31) where routing-acceptance-tests was just relying on a really old version of `rtr` instead of forcing `rtr` to be built in case there was a problem with the code-base.

The idea here is to have a recipe for building-binaries per release. This will make it so that other releases don't have to figure out how to build `rtr`. The recipe would be pulled in from the CI for other release. 

For context, diego's inigo builds `gdn` and `gorouter` and I remember fixing a bug with inigo a while back where it was still building `gdn` with runc instead of containerd.